packages/forms/src/elements/number/input.module.scss

```diff
@@ -0,0 +1,6 @@
+.percentageInput {
+  width: 75px;
+  + span {
+    position: initial;
+  }
```

Comment:
@allanthompson-tpr Too late again, sorry, but this would be better expressed in `em` units. If someone increases their text size you want the box to grow with it.